Source/WebCore/inspector/agents/InspectorCSSAgent.cpp

```diff
@@ -585,13 +591,13 @@ static Ref<Protocol::CSS::Font> buildObjectForFont(const Font& font)
             .setMaximumValue(variationAxis.maximumValue())
             .setDefaultValue(variationAxis.defaultValue())
             .release();
-        
+
```

Comment:
NIT: while I do personally like the fact that we're fixing unnecessary whitespace, it's probably best to leave as-is and fix when changing code in this area